src/stores/CheckoutController.ts

```diff
@@ -172,3 +179,43 @@ export const paypalPayment = async (successUrl: string, cancelUrl: string, error
     });
   });
 };
+
+export const getSubscriptionSwitches = async (): Promise<unknown> => {
+  const { cleengSandbox } = useConfigStore.getState().getCleengData();
+  const { user, subscription, auth } = useAccountStore.getState();
```

Comment:
Instead of using the stores/services directly, we should use the `use*` helpers like the above methods:

```suggestion
  return await useAccount(async ({ auth: { jwt } }) => {
    return await useService(async ({ checkoutService, sandbox = true, authProviderId }) => {
```

Reply:
These methods already check if the user is logged in and use the correct `checkoutService` for the configured integration.

Reply:
Thank you! I initially used the `useAccount` and `useService ` methods, but because the InPlayer service didn't contain the getSubscriptionSwitches method, I proceeded this way. But with the check that you suggested (`if (!('switchSubscription' in checkoutService))`), it now works like a charm.